docker/ubuntu/Dockerfile

```diff
@@ -53,6 +55,7 @@ RUN apt-get update && apt-get install build-essential software-properties-common
     && pip3 install typing_extensions \
     && pip3 install dataclasses \
     && pip3 install laspy \
+    && pip3 install gdown \
```

Comment:
@Mirmix Thank you for contribution!
There is already a command for `gdown` installation  (line 48).
Can you please delete this one ?